libs/cdk/schematics/src/migrations/update-1-0-0/index.spec.ts

```diff
@@ -0,0 +1,372 @@
+import { Tree } from '@angular-devkit/schematics';
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+import * as path from 'path';
+
+describe('cdk migration 1.0.0', () => {
+  let appTree: UnitTestTree;
+
+  /* Increase Jest timeout because of long living tests. */
+  beforeAll(() => jest.setTimeout(10_000));
+
+  /* Reset default timeout to 5s. */
+  afterAll(() => jest.setTimeout(5_000));
```

Comment:
:confused: I don't know if it's worth investing time to profile the migration to remove this. What do you think?

Reply:
what do you mean? to improve the tests? i won't :D